Files in /usr/local cause issues in Fedora Silverblue

## Issue report

Operating system: Fedora Silverblue 32

App version: 2020.4 (during installation)

### Issue description

When attempting to install the latest version of the app, it appears there are zsh-autocompletion files being added to /usr/local/share instead of /usr/share. This causes an issue in Silverblue and other rpm-ostree based distros due to projectatomic/rpm-ostree#233.

The issue could potentially be fixed by putting the auto completion files in /usr/share instead of /usr/local/share. I'm pretty sure this would be the convention for pre-packaged apps anyways, since /usr/local/ is usually reserved for apps not managed by the distro's package manager.

From what I can tell this is a separate issue from #1570. That issue was with files being added to /opt, which from what I understand did not exist in older versions of silverblue. However, /opt is now a symlink to /var/opt, so I didn't need to do the workaround mentioned there.

Yes you are correct, the issue with `/opt/Mullvad VPN` not existing after install is now fixed! Indeed, there is now an `/opt -> /var/opt` so this is no longer an issue :tada:.

However, there are still SELinux policy issues preventing `mullvad-daemon` being executed as a systemd service because of the `lib_t` file context, which results in SELinux permission errors when accessing the `/var/*` files:
```
$ ls -Z '/opt/Mullvad VPN/resources/mullvad-daemon'
system_u:object_r:lib_t:s0 '/opt/Mullvad VPN/resources/mullvad-daemon'
$ journalctl -u mullvad-daemon.service '/opt/Mullvad VPN/resources/mullvad-daemon' | tail -n 8
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: [mullvad_daemon::account_history][INFO] Opening account history file in /var/cache/mullvad-vpn/account-history.json
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: [mullvad_daemon::relays][DEBUG] Relay list updater thread has finished
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: [mullvad_daemon][ERROR] Error: Unable to initialize daemon
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: Caused by: Unable to load account history with wireguard key cache
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: Caused by: Unable to read account history file
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: Caused by: Permission denied (os error 13)
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: [mullvad_daemon][DEBUG] Process exiting with code 1
Jun 23 13:20:42 silverblue32 mullvad-daemon[7480]: [mullvad_daemon][INFO] Management interface shut down
```

--- a/gui/tasks/distribution.js
+++ b/gui/tasks/distribution.js
@@ -154,7 +154,7 @@ const config = {
       distAssets('linux/problem-report-link') + '=/usr/bin/mullvad-problem-report',
       distAssets('shell-completions/mullvad.bash') +
         '=/usr/share/bash-completion/completions/mullvad',
-      distAssets('shell-completions/_mullvad') + '=/usr/local/share/zsh/site-functions/_mullvad',
+      distAssets('shell-completions/_mullvad') + '=/usr/share/zsh/site-functions/_mullvad',
     ],
     afterInstall: distAssets('linux/after-install.sh'),
     afterRemove: distAssets('linux/after-remove.sh'),

